pyspecProcScripts/simple_functions.py

```diff
@@ -137,25 +137,30 @@ def determine_sign(s, direct="t2", fl=None):
 
 
 def find_apparent_anal_freq(s):
+    # PR you need a docstring here
     carrier = s.get_prop("acq_params")["carrierFreq_MHz"] * 1e6
     dt = s["t"][1] - s["t"][0]
     if carrier < 1 / dt:
         print("You are in the clear and no aliasing took place!")
         nu_a = carrier
     else:
-        SW = 2 / dt  # SW of data before made analytic - what scope sees
-        n = np.floor(  # nearest integer multiple of sampling frequency
-            (carrier + SW / 2) / SW
+        SW = 2 / dt  # SW of data before made analytic
+        #              - what scope sees
+        n = np.floor(  # nearest integer multiple of
+            #            sampling frequency.
+            #            Measured from the left side of
+            #            the shifted spectrum
+            (carrier + SW / 2)
+            / SW
         )
         nu_a = carrier - n * SW
+        isflipped = False
         if nu_a < 0:
-            # signal ended up in the negative with respect to the analytically filtered
-            # so we need to run the complex conjugate and repeat the process
-            # of finding the aliased aliased signal
-            SW = SW / 2
-            n = np.floor((carrier + SW / 2) / SW)
-            s.run(np.conj)
-            # abs ensures we calculate the correct nu_a whether the signal is aliased
-            # from below or above SW / 2
-            nu_a = abs(carrier - n * SW)
-    return s, nu_a
+            # when stored, we threw out the negative frequencies
+            # so that means that we captured an aliased copy of the
+            # negative frequency
+            nu_a = -carrier + n * SW
+            # we need to make note of the fact that the phase of our final time domain signal
+            # (after filtering and up-conversion) will be flipped
+            isflipped = True
```

Comment:
a lot of this was repeated, and this is equivalent to the abs, but only because of the conditional, so it actually makes more sense this way